### ISSUE TITLE
Some questions about the results listed in Table1 and Table6 in the paper

### ISSUE THREAD

**Issue by chen-chen32** [akhilpm/DroneDetectron2#38]
Dear Author,

In the paper, Tables 1, 2, and 6 present the model’s test results on the VisDrone dataset. Regarding these results, I have two questions:

The paper mentions, “Following the existing works, we used the validation set for evaluating the performance.” Does this imply that the results listed above are all based on the validation set, which consists of 548 pictures?
It is noted in the paper that a resolution of (1.5K pixels) was adopted. Could you please clarify what specific resolution this refers to? Additionally, could you specify the input resolutions used for the results in Tables 1, 2, and 6?
I look forward to your response and appreciate your assistance. Best regards！

**Comment by akhilpm** [comment]
Hi, Thanks for your interest in our work! To answer your questions

1. Yes, the results are reported on the validation set consisting of 548 images.
2. 1.5K pixels is the average width of images from the Visdrone Dataset. The config file for each setting has the info of the input resolution used for training and inference. See for example the [Base Faster RCNN](https://github.com/akhilpm/DroneDetectron2/blob/main/configs/Base-RCNN-FPN.yaml#L57) at inference. The images are basically resized respecting the given MIN_SIZE and MAX_SIZE constraints. Please check [Detectron2](https://github.com/facebookresearch/detectron2/blob/main/detectron2/data/detection_utils.py#L629) code for additional details.
